front/app/containers/Admin/reporting/components/ReportBuilder/TopBar/index.tsx

```diff
@@ -48,6 +48,14 @@ type ContentBuilderTopBarProps = {
   setSelectedLocale: React.Dispatch<React.SetStateAction<SupportedLocale>>;
 };
 
+// TODO remove
+const error = console.error.bind(console);
```

Comment:
Without this, the console is flooded with warnings that make it impossible to debug anything using logging. These warnings will disappear once we open a crowdin branch and the other translations files are updated. I will make sure to remove this before merging.